static/app/components/tours/components.tsx

```diff
@@ -74,27 +73,9 @@ export function TourContextProvider<T extends TourEnumType>({
     currentStepId: null,
     tourKey,
   });
-  const {dispatch, currentStepId} = tourContextValue;
+  const {currentStepId} = tourContextValue;
   const isTourActive = currentStepId !== null;
 
-  const tourHotkeys = useMemo(() => {
-    return [
-      {
-        match: 'Escape',
-        callback: () => {
-          if (tourKey) {
-            mutate({guide: tourKey, status: 'dismissed'});
-          }
-          dispatch({type: 'END_TOUR'});
-        },
-      },
-      {match: ['left', 'h'], callback: () => dispatch({type: 'PREVIOUS_STEP'})},
-      {match: ['right', 'l'], callback: () => dispatch({type: 'NEXT_STEP'})},
-    ];
-  }, [dispatch, mutate, tourKey]);
-
-  useHotkeys(tourHotkeys);
```

Comment:
If you want to add this back later, I'd propose adding a `disabled` option for `useHotKeys`